packages/flutter_tools/lib/src/compile.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert' show JSON;
+import 'dart:io' as io show File;
```

Comment:
It looks like you need to use the dart:io wrapper instead of dart:io directly. See other files for how to do this.